bin/graftM

```diff
@@ -215,8 +221,35 @@ sequence1    k__kingdom1; p__phylum2
     logging_options.add_argument('--verbosity', metavar='verbosity', help='1 - 5, 1 being silent, 5 being noisy indeed. Default = 4', type=int, default=4)
     logging_options.add_argument('--log', metavar='logfile', help='Output logging information to file', type=str, default=False)    
     
-    
-    
+    #########################################################################
+    # argparser for "decorate" - Reroot, and decorate a gene tree                           
+  
+    decorate_parser = subparsers.add_parser('decorate',
+                                            description='Reroot, and decorate a gene tree',
+                                            epilog=__author__)
+    # Main decorate options       
+    decorate_options = decorate_parser.add_argument_group('decorate options')                       
+    decorate_options.add_argument('--input_greengenes_taxonomy', type = str, help='Input taxonomy of sequences used to build tree. This taxonomy \
+must be in GreenGenes format.', default = None)
+    decorate_options.add_argument('--input_taxtastic_taxonomy', type = str, help='Input taxonomy of sequences used to build tree. This taxonomy \
+must be in taxtastic format.', default = None)
+    decorate_options.add_argument('--input_seqinfo', type = str, help='Seqinfo file to accompany Taxtastic taxonomy, the taxonomy from \
+which will be used to decorate the tree.', default = None)
```

Comment:
I'm not so sure it is. I'll otherwise the args object would have no value "input_seqinfo" causing an error when you check if a seqinfo file has been provided downstream